src/cgame/cg_players.cpp

```diff
@@ -1279,12 +1279,13 @@ static void CG_LoadClientInfo( clientInfo_t *ci )
 	// frames for this new model
 	clientNum = ci - cgs.clientinfo;
 
-	for ( i = 0; i < MAX_GENTITIES; i++ )
+	for ( int num = 0; num <= cg_highestPreviousEntityNumberInPVS; num++ )
```

Comment:
Huh, why is this even going above MAX_CLIENTS?

Reply:
I added a commit to replace the upper limit by `MAX_CLIENTS`.